src/main/webapp/WEB-INF/README.md

```diff
@@ -0,0 +1,20 @@
+## Information on the used libraries
+**Important** : All the manually-downloaded libraries which were here intially are now being installed using npm. Materialize folder is still visible here, as a download from npm, downloads all its dependencies along with the distro. This proves very costly in terms of size when war file is produced.
```

Comment:
how big is the war going to be? How much of this increase is due to materialize's package.json?